### PR TITLE
feat(annotations tooltip): Close on click outside to avoid UI issues

### PR DESCRIPTION
**Pull request by megansmith-box**

### CODE REVIEW

File: src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx [hunk 1]
```diff
@@ -16,9 +16,9 @@ function AnnotationsTargetedTooltip({ children, isEnabled = false }: Props): JSX
     return (
         <TargetedClickThroughTooltip
             className="bp-AnnotationsTooltip"
+            closeOnClickOutside
             shouldTarget={isEnabled}
             showCloseButton
-            tetherElementClassName="bp-AnnotationsTooltipTether"
```
Comment (jstoffan) [inline src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx:21]: Just to confirm: Is this change intentional? Is the increased z-index no longer needed?

**Review by jstoffan — APPROVED** [review]

File: src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx (hunk 1 above)
Comment (megansmith-box) [inline src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx:21]: we no longer need this as the tooltip will close if the preview share modal is clicked

Comment (megansmith-box) [reply src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx:21]: @jstoffan - Correct, we no longer need the z-index, because if we click on the share button which opens the modal the tooltip will now close.
